content/android-privacy-policy.md

```diff
@@ -1,15 +1,15 @@
 # Syncthing-Android Privacy Policy
 
-Read this document to understand how the app Syncthing-Android uses data.
+Read this document to understand how the app Syncthing-Android uses data and respects user privacy.
 
 ---
 
-Syncthing-Android does not collect any user information.
+Syncthing-Android values user privacy and thus does not collect any user information.
 
-The app does not store any location data.
+For location data includes very sensitive personal information related to user privacy, the app does not store any location data.
 The permission to access location in the background is used only to check for an SSID of the currently active Wi-Fi.
 The sole purpose of this information is to limit synchronisation to user-configured Wi-Fi connections.
-The app itself never collects the location in the process, and as such it also does not use or save any location data.
+At no point is this information used to look up your actual location nor is it ever passed on to 3rd parties.
```

Comment:
At no point is this information used to track the user's actual location nor is it ever shared with any 3rd party entities.

Reply:
What's the problem with "look up"? "Track" implies something more to me, something continuous. What it should say is that we don't look at their location, not even once.

Reply:
I was looking for a more formal word, but nothing better came to my mind. "Determine" is much better!